cloudSupport/src/main/scala/cromwell/cloudsupport/gcp/auth/GoogleAuthMode.scala

```diff
@@ -81,14 +71,26 @@ sealed trait GoogleAuthMode {
 
   def requiresAuthFile: Boolean = false
 
+  // changing this is for testing only. all the inheritance is otherwise locked down.
```

Comment:
I think I understand what this sentence is saying but I'm not sure, which makes me think it should be rephrased somehow